p4-16/spec/P4-16-spec.mdk

```diff
@@ -6796,12 +6823,14 @@ order, stopping at the first matching entry. Architectures should
 define the significance of entry order (if any) for other kinds of
 tables.
 
-Depending on the `match_kind` of the keys, key set expressions may define
-one or multiple entries. The compiler will synthesize the correct number of
-entries to be installed in the table. Target constraints may further restrict
-the ability of synthesizing entries. For example, if the number of synthesized
-entries exceeds the table size, the compiler implementation may choose to issue
-a warning or an error, depending on target capabilities.
+Depending on the `match_kind` of the keys, key set expressions may
+define one or multiple entries, as described in Section
+[#sec-table-keys]. The compiler will synthesize the correct number of
```

Comment:
I am not fundamentally against adding this phrase "as described in Section [#sec-table-keys}" here, but I do not see anywhere in that section that specifies how one key set expression may turn into one or multiple entries in a table.  It seems reasonable to me that the language specification NOT try to specify that level of detail.

Basically, the addition of this phrase seems confusing to me.  If I go where it says to go, it does not explain what the sentence before it claims it will.

Reply:
My previous comment still seems not addressed.

Reply:
The cross-reference has been removed, addressing my comment.